setup.py

```diff
@@ -11,12 +11,12 @@
     license='MIT',
     description='Very flexible date parsing and normalization utilities',
     long_description=__long_description__,
-    author='Open Knowledge Foundation',
```

Comment:
In general i would suggest this be left with Open Knowledge as primary author for purposes of metadata for PyPI. I'd suggest adding additional contributor and author info in the README for the present..